src/PSBrowseWrap.js

```diff
@@ -31,15 +31,6 @@ class PSBrowseWrap extends React.Component {
     _ps.getByKey(groupKey).rendered = false;
   }
 
-  isSnippetLoaded() {
```

Comment:
thought here: we should spin up PI top remove browswrap from this package

Reply:
Will do!